draft-ietf-taps-transport-security.md

```diff
@@ -495,27 +493,38 @@ the RTP SSRC (synchronization source), packet index, and session "salting key".
 SRTP offers replay detection by keeping a replay list of already seen and processed packet indices. 
 If a packet arrives with an index that matches one in the replay list, it is silently discarded.
 
-DTLS {{RFC5764}} is commonly used as a way to perform mutual authentication and key 
-agreement for SRTP {{RFC5763}}. (Here, certificates marshal public keys between
-endpoints. Thus, self-signed certificates may be used if peers do not mutually trust one another, 
-as is common on the Internet.) When DTLS is used, certificate fingerprints are transmitted
-out-of-band using SIP. Peers typically verify that DTLS-offered certificates match
-that which are offered over SIP. This prevents active attacks on RTP, but not on the signaling (SIP or
-WebRTC) channel. 
+DTLS {{RFC5764}} is commonly used to perform mutual authentication and key
+agreement for SRTP {{RFC5763}}.
+Peers use DTLS to perform mutual certificate-based authentication on the
+media path, and to generate the SRTP master key.
+Peer certificates can be issued and signed by a certificate authority.
+Alternatively, certificates used in the DTLS exchange can be self-signed.
+If they are self-signed, certificate fingerprints are included in the signalling
+exchange (e.g., in SIP or WebRTC), and used to bind the DTLS key exchange in
+the media plane to the signaling plane.
+The combination of a mutually authenticated DTLS key exchange on the media
+path and a fingerprint sent in the signalling channel will protect against
+active attacks on the media, provided the signalling can be trusted.
+The signalling needs to be protected as described in, for example, SIP
+{{RFC3261}} Authenticated Identity Management {{RFC4474}} or the WebRTC
+security architecture {{I-D.ietf-rtcweb-security-arch}}, to provide
+complete system security.
+
+
 
 ### Protocol features
 
-- Optional replay protection with tunable replay windows.
-- Out-of-order packet receipt.
-- (RFC5763) Mandatory mutually authenticated key exchange.
 - Partial encryption, protecting media payloads and control packets but not data packet headers.
 - Optional authentication of data packets; mandatory authentication of control packets.
+- Optional replay protection with tunable replay windows.
+- Out-of-order packet receipt.
+- Mutually authenticated key exchange {{RFC5764}}
 
 ### Protocol dependencies
 
-- External key derivation and management mechanism or protocol, e.g., DTLS {{RFC5763}}.
-- External signaling protocol to manage RTP parameters and locate and identify peers, e.g., 
-SIP {{RFC3261}} or WebRTC {{I-D.ietf-rtcweb-security-arch}}.
+- External key derivation and management protocol, e.g., DTLS {{RFC5763}}.
+- External identify management protocol, e.g., SIP Authenticated Identify Management
```

Comment:
s/identify/identity, and same for "Identify"

Reply:
Spell checkers are rubbish :)